torch/_export/passes/add_runtime_assertions_for_constraints_pass.py

```diff
@@ -51,6 +56,41 @@ def _convert_range_to_int(range: RangeConstraint):
 
 
 class _AddRuntimeAssertionsForConstraintsPass(ExportPassBase):
+    @classmethod
+    def update_exported_program_signature(
```

Comment:
Let's move this back to exported_program.py. I feel little uneasy about exposing this as a class method and introducing a circular dependency. As a follow up, we will discuss what the proper way to abstract the passes that can work with ExportedProgram.